AMIA 2016 Annual Symposium
# Mining Large-scale Cancer Genomics Data Using Cloud-based Bioinformatics Approaches   

Working Group Pre-symposium WG13   
8:30 AM–12:00 PM Nov 13, 2016   
Boulevard A   

[Center for Research Informatics](https://cri.uchicago.edu/), The University of Chicago   
Session link: https://amia2016.zerista.com/event/member/287281     

**Description**    
Abstract: Next-generation sequencing (NGS) is now routine in cancer research, genetic testing, and the application of precision medicine. Despite the proliferation of tools and the commoditization of high-performance computing, it remains a challenge to transform the enormous amounts of sequencing data into meaningful biological information. While a single run of ultra-high throughput sequencing could generate 6 billion reads covering 20 whole human genomes, analysis of these data requires advanced computational skills, access to high-performance computing infrastructure, costly commercial software, and specialized consultants. In this AMIA Genomics and Translational Bioinformatics Working Group pre-symposium, we provide guidelines and project-oriented hands-on training on the key components of analyzing multi-dimensional cancer genomics data. Experienced bioinformaticians and computational biologists will guide the participants through the analysis of RNA-Seq and ChIP-Seq data and the visualization of results using Amazon’s AWS EC2 as a virtual infrastructure. Participants will gain knowledge in experimental design and sample collection and experience with popular bioinformatics tools for data preprocessing, alignment, quantification, differential analysis, and visualization. Participants will leave the workshop with the skills to start analyzing their own datasets using cloud infrastructure.

**Learning Objective 1**: Understand the principles, variables, considerations, and limitations in designing NGS experiments for human studies and obtain a deeper knowledge of when and how to apply NGS technologies in biomedical research

**Learning Objective 2**: Learn how to analyze RNA-Seq data, ChIP-Seq data, how to assess sample and data quality and how to determine if an experiment is successful

**Learning Objective 3**: Learn standard NGS file formats and best-practice analysis workflows and how to generate and customize publication-quality plots for data visualization

**Learning Objective 4**: Be motivated to establish communications with informatics experts and other participants for future collaborations

# Useful Links
Lecture, pipelines and hands-on materials of RNAseq/ChIPseq data analysis are provided at
* [CRI-Workshop-AMIA-2016-RNAseq](https://github.com/cribioinfo/CRI-Workshop-AMIA-2016-RNAseq)
* [CRI-Workshop-AMIA-2016-ChIPseq](https://github.com/cribioinfo/CRI-Workshop-AMIA-2016-ChIPseq)
